websrc/source/documents/tutorials/contiv-compose.md

```diff
@@ -6,33 +6,42 @@ description: |-
   Getting Started
 ---
 
-# Policies with Networking
+## Networking Policies with Compose
 
 This tutorial shows how to use a modified *libcompose* utility to apply network policies on a Docker application composition.
 
-**Note:** The demonstrations on this page use the Vagrant utility to set up a VM environment. This environment is for demonstrating automation and integration with Contiv and is not meant to be used in production.
+### Prerequisites 
+1. [Download Vagrant](https://www.vagrantup.com/downloads.html)
+2. [Download Virtualbox](https://www.virtualbox.org/wiki/Downloads)
+3. [Install git client](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+4. [Install docker for mac](https://docs.docker.com/docker-for-mac/install/)
 
-## Getting Started
-The following steps describe how to set up a demo application and apply policies to it.
+**Note**:
+- If you are using platform other than Mac, please install docker-engine, for that platform.
 
-### Prerequisites
-Before starting, install the following tools on your Linux or OS X machine:
 
-- Make
-- Git
+### Setup
 
-### 1. Start the Vagrant Environment
-Use the following commands to start a Contiv Vagrant setup.
+#### Step 1: Get contiv installer code from github.
+```
+$ git clone git@github.com:contiv/install.git
```

Comment:
Can you use the https URL here?  `git@...` is SSH which requires a github account + ssh keypair setup.  https will work for anyone